.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,88 @@
+name: Bug Report
+description: Report a reproducible bug or regression in Joplin.
+labels: ['type: bug']
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      options:
+        - label: I have used the search function for [open](https://github.com/laurent22/joplin/issues) **and** [closed](https://github.com/laurent22/joplin/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
+          required: true
+        - label: I will describe the problem with as much detail as possible.
+          required: true
+        - label: I will update Joplin to the latest version before reporting a bug.
+          required: true
+        
+  - type: input
+    id: version
+    attributes:
+      label: Joplin version
+      description: We need the actual version number found in the `About Joplin` menu.
+      placeholder: x.y.z
```

Comment:
For desktop it would be less error prone to copy the whole contents of `About Joplin`. This way it would also show plugins installed which might be causing problems.

For mobile the version number is located at the bottom of `Configuration`